### PR TITLE
fix(web-components): added margin left to mobile app container

### PR DESCRIPTION
**Pull request by za2201**
## Summary of the changes
added margin left to app title container on mobile to fit focus state

## Related issue
#540 

## Checklist
- [x] I have added relevant unit and visual regression tests.
- [ ] I have manually accessibility tested any changes, if relevant.
- [ ] I have ensured any changes match the Figma component library.

### CODE REVIEW

File: packages/web-components/src/components/ic-top-navigation/ic-top-navigation.css [hunk 1]
```diff
@@ -245,6 +245,10 @@ slot[name="toggle-icon"] svg {
     min-height: 2.5rem;
   }
 
+  .app-details-container {
+    margin-left: var(--ic-space-xs);
```
Comment (GCHQ-Developer-112) [inline packages/web-components/src/components/ic-top-navigation/ic-top-navigation.css:249]: The app details container was designed to be positioned 8px from the left in mobiles. This change will mean that the extra small top navigation will not look as the designers intend

**Review by GCHQ-Developer-112 — CHANGES_REQUESTED** [review]
This fix affects the design of the extra small top nav. Please could you try another fix that doesn't affect the padding/margins in the top nav

File: packages/web-components/src/components/ic-top-navigation/ic-top-navigation.css (hunk 1 above)
Comment (GCHQ-Developer-112) [reply packages/web-components/src/components/ic-top-navigation/ic-top-navigation.css:249]: Please ignore the comment above. I've spoken to the design team who are happy with pushing the app title to the right 4px

**Comment by jd239** [comment]
Slight update for this ticket. We have noticed a negative margin-left which should be removed instead of the additional styling added as part of the ticket: https://github.com/mi6/ic-ui-kit/blob/develop/packages/web-components/src/components/ic-top-navigation/ic-top-navigation.css#L24

**Review by GCHQ-Developer-112 — DISMISSED** [review]

**Review by jd239 — APPROVED** [review]

**Review by GCHQ-Developer-847 — APPROVED** [review]

**Review by GCHQ-Developer-112 — APPROVED** [review]

**Pull request merged**